app/models/guest.rb

```diff
@@ -2,4 +2,8 @@ class Guest
   def company_owner?
     false
   end
+
+  def is_admin?
```

Comment:
```suggestion
  def is_admin? # rubicon:disable Naming/PredicateName
```

А ще краще ` def admin?` і відповідну колонку переіметнувати у `admin`ю

Reply:
Поправив suggestion у попередньому коментарі.
А колонку мабуть краще у якомусь іншому ПРі переіменувати.

Reply:
Я уже її перейменував, відмінити зміни?